### PR TITLE
Replace Material Icons by Angular Font Awesome Icons

### PR DESCRIPTION
**Pull request by love-opensource**
## Description
I have replaced the material icons by the Angular Font Awesome icons in Clients, Groups and Centers components. 
This is because all components are being refactored to use angular font awesome icons so as to remove the material icons dependencies.

## Related issues and discussion
#138

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by abhaychawla** [comment]
@love-opensource Again, you have not followed the pull request template. The commit message is not according to the guidelines. And instead of correcting the same PR you have sent a separate PR. Could you please follow the conventions? If you have doubts please ask.

**Comment by love-opensource** [comment]
@abhaychawla please where can I find the pull request template? 
I'll have to close this one and go back and correct the previous one.
Thanks.

**Comment by abhaychawla** [comment]
The content of the template is shown to you when you send the PR. You need to fill the details in it. I guess you removed it. You can have a look at `.github` folder of the repository to find the same now.

**Comment by love-opensource** [comment]
Ok. Since the last pull request was made with two github accounts, I'll like to follow on this one and close the other one.
Thanks.

File: src/app/centers/centers.component.html [hunk 1]
```diff
@@ -29,9 +29,11 @@
       <ng-container matColumnDef="status">
         <th mat-header-cell *matHeaderCellDef mat-sort-header='status'> Status </th>
         <td mat-cell *matCellDef="let row"> <div [className]="row.active === true ? 'active' : 'inactive'">
-            <i class="fa fa-circle fa-2x"> </i>
+            <mat-icon matListIcon>
```
Comment (abhaychawla) [inline src/app/centers/centers.component.html]: Please update all icons in table in a similar manner to how its being used in `chart-of-accounts.component.ts` of the accounting section. Here is an example for your reference:
```html
<div [className]="row.active === true ? 'active' : 'inactive'">
    <fa-icon matTooltip="{{ row.active === true ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="circle" size="lg"></fa-icon>
</div>
```

**Comment by love-opensource** [comment]
@abhaychawla I resolved the issue. How should I integrate the branch from my fork repo to this pull request?

**Comment by abhaychawla** [comment]
@love-opensource Any changes you make to the branch on your fork via which you have sent this PR are automatically reflected here as you can see. Please squash your commits into a single commit. Remove the navigation component file from your commit and any redundant code that has been left out.

**Comment by abhaychawla** [comment]
@love-opensource It has been a long time since you have been inactive. Shall I expect an update on this PR or close it so any other contributor might work on it?

**Comment by abhaychawla** [comment]
Closing this PR due to a long period of inactivity.

**Pull request closed without merging**